client/src/index.scss

```diff
@@ -38,6 +38,10 @@ main {
     @include minQ(768px) {
         margin-left: 110px;
     }
+    &.unscrollable {
+        overflow: hidden; 
+        position: fixed;
+    }
```

Comment:
As seen on the picture attached to the PR, the <main> gets squashed when you open the hamburger menu. That is not the expected behaviour, we want it to stay 100% wide.

Reply:
Done, thank you!